test/fuse/linux/mknod_test.cc

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <errno.h>
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <sys/statfs.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#include <linux/fuse.h>
+
+#include <string>
+#include <vector>
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "test/util/test_util.h"
+#include "test/util/fuse_util.h"
+#include "test/util/fs_util.h"
+
+#include "fuse_base.h"
+
+namespace gvisor {
+namespace testing {
+
+namespace {
+
+class MknodTest : public FuseTest {};
+
+TEST_F(MknodTest, RegularFile) {
+  const std::string testFileName = "testFile";
+  const std::string testFilePath = JoinPath(kMountPoint, testFileName);
```

Comment:
Use [underscore](https://google.github.io/styleguide/cppguide.html#Variable_Names) instead of CamelCase in variable. And you can declare variables in class `MknodTest`'s protected area if they are the same across different tests.